app/content/prestaking.js

```diff
@@ -1,7 +1,7 @@
 export default {
   hero: {
-    title: 'Pre-stake to earn your share of 200 Million NIM in rewards and bonus giveaways!',
-    body: 'Join in and support the network. Be among the first to earn, get a unique Identicon upgrade and win some NIM.',
+    title: 'Pre-staking is closed. Get Ready for What’s Next!',
+    body: 'Thank you to everyone who joined in and supported the network. If you pre-staked, thanks! You ll receive your rewards directly in your wallet within the next 30 days.',
```

Comment:
```suggestion
    body: 'Thank you to everyone who joined in and supported the network. If you pre-staked, thanks! You'll receive your rewards directly in your wallet within the next 30 days.',
```

Reply:
And I thought it's 3 months, not 30 days?

Reply:
I think it is 3-months. Yes. I had written (insert proper duration) in the notes. Thanks.